core/nodebase.cpp

```diff
@@ -330,7 +330,7 @@ bool NodeBase::setIntervalRequest(const int sessionId, const unsigned int interv
     }
 
     // Store the request for the session
-    m_intervalMap[sessionId] = interval_us;
+    m_intervalMap[sessionId] = validatedInterval_ms;
```

Comment:
This change should be in next commit "Use closest supported data rate". Also remove the extra empty line change below from this commit.

Reply:
yes, completely missed this, will fix